### ISSUE TITLE
02-numpy: use patient data for 1D array in Change In Inflammation exercise 

### ISSUE THREAD

**Issue by ldko** [swcarpentry/python-novice-inflammation#780]
The exercise _Change In Inflammation_ in the _Analyzing Patient Data_ episode could better blend with the overall lesson story by using the actual patient data when explaining the purpose of `numpy.diff` and giving the one-dimensional array example. For example, the one-dimensional array could be patient 3's first week of data as [suggested by Maxim here](https://github.com/swcarpentry/python-novice-inflammation/pull/777#issuecomment-579982232).

We might also tweak the text of the question:
```
When applying numpy.diff to our 2D inflammation array data, which axis would it make
sense to use this function along?
```
to first indicate you can pass an `axis` argument for something like:
```
When calling `numpy.diff` with a multi-dimensional array, an `axis` argument may be
supplied to specify for which axis to take the differences. When applying
`numpy.diff` to our 2D inflammation array data, which axis would we specify?
```

**Comment by noatgnu** [comment]
Hi @ldko  , would these changes be sufficient?

> The `numpy.diff()` function takes an array and returns the differences
> between two successive values. We would like to use it to examine the changes
> each day across the first week of patient 3 from our inflammation dataset.
> 
> ~~~
> row_start = data[3, :7] 
> ~~~
> {: .language-python}
>
> Calling `numpy.diff(row_start)` would do the following calculations
>
> ~~~
> [ 0 - 0, 2 - 0, 0 - 2, 4 - 0, 2 - 4, 2 - 2 ]
> ~~~
> {: .language-python}
>
> and return the 6 difference values in a new array.
>
> ~~~
> numpy.diff(row_start)
> ~~~
> {: .language-python}
>
> ~~~
> array([ 0,  2, -2,  4, -2,  0])
> ~~~
> {: .output}
>
> Note that the array of differences is shorter by one element (length 6).
>
> When calling `numpy.diff` with a multi-dimensional array, an `axis` argument may
> be passed to the function to specify which axis that the function would process.
> When applying `numpy.diff` to our 2D inflammation array `data`, which axis
> would we specify?

**Comment by ldko** [comment]
Hi Toan, Those are generally the changes I think we are looking for. I also liked the suggestion by @maxim-belkin to rename `row_start` to `patient3_week1` to hopefully make it clear what the value means. I would also tweak the wording of "We would like to" to "Let's" and perhaps also "to specify which axis that the function would process" to "to specify the axis to process". When you have time, please open a pull request, and if the other maintainers have any other suggestions we can get them there. Thanks for picking up this issue!

**Comment by ldko** [comment]
Fixed in #782
